
public class PruebaSinException {
	public static void main(String[] args) throws Exception {
		try {
			System.out.println("Hola");
			throw new Exception();
		}
		catch (Exception e){
		
		}
		finally{
			
		}
		System.out.println("Hola 2");
	}
}
